opensrp-core/src/main/java/org/smartregister/service/UserService.java

```diff
@@ -848,4 +849,10 @@ public byte[] getGroupId(String userName, KeyStore.PrivateKeyEntry privateKeyEnt
         }
         return null;
     }
+
+    public void saveUserId(String userName, String baseEntityId) {
+        if (userName != null) {
+            allSharedPreferences.saveUserId(userName, baseEntityId);
+        }
+    }
```

Comment:
Not sure what we are using the `user ID` for

Reply:
these also came from https://github.com/opensrp/opensrp-client-core/pull/887/files